### ISSUE TITLE
Xmas holidays commission.

### ISSUE THREAD

**Issue by Irostovsky** [LjaljevicStefan/GetCommissionDatetime#1]
Hello @LjaljevicStefan 
I m Ivan, from GuestReady.
Thanks for the doing task.
Just one question. Imagine that we want to have the 12% commission in Dec and Jan because of Xmas holidays. How the code should be modified?

**Comment by LjaljevicStefan** [comment]
Hello @Irostovsky
Changes for Xmas holidays commission has been submitted.

**Comment by Irostovsky** [comment]
@LjaljevicStefan thanks for the update.
Unfortunately we can’t go forward with proposed solution.
The expected one would be:
Define a dict with month_number: commission. Ie
{1: 0.12,
2: 0.1,
….
}
then just get the commission from dict by month of the passed date.
The solution fixes 2 major issues: there is no need to change a function code when the commission is changed for some particular month.
And preventing magic numbers anti-pattern in code. So data and logic are splitted.

**Comment by LjaljevicStefan** [comment]
Thank you for getting back to me, that solution didn't occur to me because
in first task it didn't mention that there would be a lot of changes on
commission factor, but now when you mention dict, it is more logical
solution.

**Comment by Irostovsky** [comment]
Yep, the task was with this small trick especially. Then I have given the tip(Xmas) that should help to resolve it in a more less right way.
